stable/database/values.yaml

```diff
@@ -20,6 +20,7 @@ cloud:
     # If the NuoDB domain spans 2 or more physical clusters, then each cluster must have a unique cluster.name.
     # The default is fine for single-cluster domains.
     name: cluster0
+    entrypointName: cluster0
```

Comment:
Do we need this?

Reply:
Check with Nik and Aaron,  this is so that when using multi-clusters you can peer to admin layer already started in a different cluster. I think it should be more generic to peer with already admin.   As in admin started in namespace a and want to start in namespace b.    So make it a fully qualified name.

Reply:
Dirk is right that is what it's for but I think stanimir has added here only because it is referenced in helpers.  namespace a/b scenario (in one cloud) is handled with the existing admin.namespace value I believe (so provide admin.namespace in separate database values file).  What we can't currently configure is peering an admin in one NS2/cloud2 to admin in NS1/cloud1, only NS1/cloud2 to NS1/cloud1.

Reply:
Yes. I added this because it's referenced in the chart. Technically it's not needed because it's defaulted in the templates but I think that it's good practice to have it in the values so users know that it's used.

@acabrele I'm not sure I understand completely the rest of your comment about and is it related to this PR or is more general limitation.

Reply:
More general response to Dirk, no additonal change requested.

Reply:
Can we deploy databases in one NS and admin in another NS.   I have no problem with admin pretty much always being in NS - nuodb.   However,  I can see projects deploying in another NS using the same admin if possible.  This was all possible a long time ago when we first wrote the helm charts. or whatever OpenShift had back then.

Reply:
@butson Yes this is possible, however since we added the serviceAccount, it requires some additional effort (before that, it worked without any modifications).  The issue being simply that the SA and role are created during admin deployment in the admin NS.  The db deployment in a separate DB NS is then missing the SA and role.  So this could be worked around by either setting the SA to be _default_ in values instead of _nuodb_ if security allows (like we used to), or manually deploying the SA/Role in the DB namespace.